lib/jazzy/config.rb

```diff
@@ -247,18 +247,30 @@ def expand_path(path)
                     'Example: http://git.io/v4Bcp'],
       default: []
 
+    config_attr :theme_directory,
```

Comment:
Note that this will make it be called “theme_directory” instead of “theme” in config files, even though it will take the same value. Might be confusing.